sdks/java/harness/src/main/java/org/apache/beam/fn/harness/data/MultiplexingFnDataReceiver.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.beam.fn.harness.data;
+
+import java.util.Collection;
+import org.apache.beam.fn.harness.fn.ThrowingConsumer;
+import org.apache.beam.sdk.fn.data.FnDataReceiver;
+
+/**
+ * A {@link FnDataReceiver} which forwards all received inputs to a collection of {@link
+ * ThrowingConsumer consumers}.
+ */
+public class MultiplexingFnDataReceiver<T> implements FnDataReceiver<T> {
+  public static <T> MultiplexingFnDataReceiver<T> forConsumers(
```

Comment:
Return a FnDataReceiver and not a multiplexing one.

Also, for the case where there is only one FnDataReceiver in the collect, return the FnDataReceiver itself.

Reply:
Done.